hoodie-client/src/main/java/com/uber/hoodie/WriteStatus.java

```diff
@@ -47,12 +48,14 @@
     private long totalRecords = 0;
     private long totalErrorRecords = 0;
 
-    public void markSuccess(HoodieRecord record) {
+    public void markSuccess(HoodieRecord record,
+        Map<String, String> recordMetadata) {
```

Comment:
Make WriteStatus a interface and have a Default implementation

Reply:
Sure, Im planning to have the default implementation also not use recordMetadata. Please let me know if you prefer otherwise.